src/accounts/page.rs

```diff
@@ -15,27 +15,28 @@ use crate::transactions::components::*;
 #[component]
 pub fn AccountSummaryPage() -> impl IntoView {
     let memo_params_map = use_params_map();
-    let public_key = memo_params_map
-        .with(|params| params.get("id").cloned())
-        .unwrap_or_default();
 
-    let resource: Resource<(), Result<AccountResponse, MyError>> = {
-        create_resource(
-            || (),
-            move |_| {
-                let public_key_for_async = public_key.clone();
-                async move { load_data(&public_key_for_async).await }
-            },
-        )
-    };
+    let resource = create_resource(
+        move || memo_params_map.get(),
+        |value| async move {
+            if let Some(id) = value.get("id").cloned() {
+                let id_clone = id.clone();
```

Comment:
Nit: This clone seems redundant since you've already cloned the value in the `if`. This shouldn't block the merge however.